README.md

```diff
@@ -233,6 +233,22 @@ If you want to enable Syntactic & Semantic TypeScript error reporting you can en
 }
 ```
 
+### Ignore coverage on decorators
+If you want to ignore coverage on decorators you can enable this through `ignoreCoverageForDecorators` and `ignoreCoverageForAllDecorators` flags. If you enable the first option you have to add the `/* istanbul ignore decorator */` comment after the decorator. If you choose the second option all decorators will be ignored.
+
+```json
+{
+  "jest": {
+    "globals": {
+      "ts-jest": {
+        "ignoreCoverageForDecorators": true,
```

Comment:
I think the name for this configuration option is a bad choice. I'd recommend something such as `enableDecoratorIgnoreComment`. Maybe this should simply be `true` always and not configurable.

Reply:
As this is basically a fix for using typescript with istanbul I think both should default to `true`.